File: fabric-frontend/src/app/api/models/snapshot-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SnapshotComponentDto } from './snapshot-component-dto';
import { SnapshotId } from './snapshot-id';
export interface SnapshotDto {
  componentSnapshots?: Array<SnapshotComponentDto>;
  environmentKey?: string;
  name?: string;
  snapshotId?: SnapshotId;
}

File: fabric-backend-core/src/main/java/io/storydoc/fabric/snapshot/app/result/SnapshotDTO.java
package io.storydoc.fabric.snapshot.app.result;

import io.storydoc.fabric.snapshot.domain.SnapshotId;
import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@Builder
public class SnapshotDTO {
    private String name;
    private String environmentKey;
    private SnapshotId snapshotId;
    private List<SnapshotComponentDTO> componentSnapshots;

}
